src/components/Layout/Footer/index.tsx

```diff
@@ -0,0 +1,43 @@
+/** @jsxImportSource @emotion/react */
+import styles from "./styles";
+import { NavLink } from "react-router-dom";
+import {
+  FacebookIcon,
+  TwitterIcon,
+  YoutubeIcon,
+  InstagramIcon,
+} from "./footerIcons/Icons";
+import "../../../styles/common.css";
+
+const Footer = () => {
+  const icons = [
+    { component: FacebookIcon, id: 0 },
+    { component: TwitterIcon, id: 1 },
+    { component: YoutubeIcon, id: 2 },
+    { component: InstagramIcon, id: 3 },
+  ];
```

Comment:
💊 
렌더링에 영향을 미치지 않는 변수네요! 매 렌더마다 새로 icons가 생성 될 필요가 없어보여서 이렇게 빼는건 어떨까요 👀 
```suggestion
const ICON_LIST = [
  { component: FacebookIcon, id: 0 },
  { component: TwitterIcon, id: 1 },
  { component: YoutubeIcon, id: 2 },
  { component: InstagramIcon, id: 3 },
];
  
const Footer = () => {
```